devito/compiler.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, **kwargs):
             self.cc = self.MPICC if kwargs.get('cpp', False) is False else self.MPICXX
         self.ld = self.cc  # Wanted by the superclass
 
-        self.cflags = ['-O3', '-g', '-fPIC', '-Wall', '-std=c99']
+        self.cflags = ['-O3', '-g', '-fPIC', '-Wall', '-std=c99', '-pedantic']
```

Comment:
I see why you added it, but do we really _need_ it? pragmatically speaking...

Did you check this is not causing any of the GCC compilers to emit warnings we would never want users to see? I think it really shouldn't since `pedantic`, if anything, should throw errors not warnings...

Reply:
dropped